packages/fxa-payments-server/src/config.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  CONTENT_SERVER_ROOT: "http://127.0.0.1:3030",
+  PROFILE_API_ROOT: "http://127.0.0.1:1111/v1",
+  OAUTH_API_ROOT: "http://127.0.0.1:9010/v1",
+  AUTH_API_ROOT: "http://127.0.0.1:9000/v1",
+  STRIPE_API_KEY: "pk_test_FL2cOisOukoCQUZsrochvTlk00ff4IakfE",
```

Comment:
FWIW, this is a testing API key from my personal account. Only use [testing card numbers from Stripe, here](https://stripe.com/docs/testing#cards).

Reply:
I guess test instructions should go into the README